test/Contract/Minting.purs

```diff
@@ -0,0 +1,81 @@
+module Test.Contract.Minting (suite) where
+
+import Contract.Prelude
+
+import Contract.Address (getWalletAddress, scriptHashAddress)
+import Contract.Chain (currentSlot)
+import Contract.Monad (liftContractE, liftedM)
+import Contract.Test.Plutip (runPlutipContract, withKeyWallet, withStakeKey)
+import Data.BigInt as BigInt
+import Mote (test)
+import Seabug.Contract.Util (getSeabugMetadata)
+import Seabug.MarketPlace (marketplaceValidatorAddr)
+import Seabug.Types (LockDatum(..), MarketplaceDatum(..))
+import Test.Contract.Util
+  ( assertContract
+  , assertTxHasMetadata
+  , callMintCnft
+  , callMintSgNft
+  , checkNftAtAddress
+  , assertOutputHasDatum
+  , findUtxoWithNft
+  , mintParams1
+  , plutipConfig
+  , privateStakeKey1
+  )
+import TestM (TestPlanM)
+
+suite :: TestPlanM Unit
+suite =
```

Comment:
Is there scope for any failing txs? I'd like to include at least one for each flow

Reply:
Included failing tx tests for minting, and made an issue for the remaining #52